materialize-databricks/.snapshots/TestValidateAndApplyMigrations

```diff
@@ -85,6 +85,6 @@ Migratable Changes Applied Schema:
 
 
 Migratable Changes Applied Data:
-key (STRING), _meta/flow_truncated (BOOLEAN), boolWidenedToJson (BOOLEAN), flow_published_at (TIMESTAMP), int64 (BIGINT), multiple (STRING), nonScalarValue (STRING), optional (STRING), scalarValue (STRING), stringWidenedToJson (STRING), timeValue (STRING), flow_document (STRING), dateValue (STRING), datetimeValue (STRING), intWidenedToJson (STRING), numericString (STRING), requiredNumeric (STRING)
-1, false, true, 2024-09-13T01:01:01Z, 1, <nil>, <nil>, <nil>, test, hello, 01:01:01, {}, 2024-01-01, 2024-01-01T01:01:01.111111000Z, 999, 123, 456
+key (STRING), _meta/flow_truncated (BOOLEAN), boolWidenedToJson (BOOLEAN), flow_published_at (TIMESTAMP), multiple (STRING), nonScalarValue (STRING), optional (STRING), scalarValue (STRING), stringWidenedToJson (STRING), timeValue (STRING), flow_document (STRING), dateValue (STRING), datetimeValue (STRING), int64 (DECIMAL), intWidenedToJson (STRING), numericString (STRING), requiredNumeric (STRING)
```

Comment:
For this and some of the other snapshots, the ordering of columns appears to have changed and I'm not 100% sure what that is, but it shouldn't make any difference in how the connector operates.